lms/djangoapps/verify_student/tests/test_services.py

```diff
@@ -167,6 +178,26 @@ def test_get_verify_location_from_string(self):
         expected_path = f'{settings.ACCOUNT_MICROFRONTEND_URL}/id-verification'
         assert path == (expected_path + '?course_id=course-v1%3AedX%2BDemoX%2BDemo_Course')
 
+    @override_settings(
+        OPEN_EDX_FILTERS_CONFIG={
+            "org.openedx.learning.idv.page.url.requested.v1": {
+                "pipeline": [
+                    "lms.djangoapps.verify_student.tests.test_services.TestIdvPageUrlRequestedPipelineStep",
+                ],
+                "fail_silently": False,
+            },
+        },
+    )
+    def test_get_verify_location_with_filter_step(self):
+        """
+        Test IDV flow location can be customized with an openedx filter
+        """
+        url = IDVerificationService.get_verify_location()
+        assert url == TestIdvPageUrlRequestedPipelineStep.TEST_URL
+
+        url = IDVerificationService.get_verify_location('course-v1:edX+DemoX+Demo_Course')
```

Comment:
Suggestion: is there a reason that the urls tested are both the same here? I'm not sure if this would make a difference, but would it be better for the urls to be different. Or is the point that they should in fact be the same? Maybe I'm misunderstanding.

Reply:
I want the output the same because I'm validating the url is overridden by the filter hook in both cases.

Reply:
We should also test the case when the filter is not configured so we know it's not getting overridden.

Reply:
yup! The existing tests above this one cover the default behavior with no filter configured